src/app/features/characters/characters.component.ts

```diff
@@ -0,0 +1,29 @@
+import { CommonModule } from '@angular/common';
+import { Component } from '@angular/core';
+import { Observable } from 'rxjs';
+
+import { RandomCharacterComponent } from './components/random-character/random-character.component';
+import { CharacterCardComponent } from './components/character-card/character-card.component';
+import { MarvelService } from './services/marvel.service';
+import { CharacterInfoComponent } from './components/character-info/character-info.component';
+
+@Component({
+  selector: 'app-characters',
+  standalone: true,
+  imports: [ CommonModule, RandomCharacterComponent, CharacterCardComponent, CharacterInfoComponent ],
+  templateUrl: './characters.component.html',
+  styleUrl: './characters.component.scss'
+})
+export class CharactersComponent {
+  list$: Observable<any[]> = this.marvelService.getCharacters();
```

Comment:
such kind of initializations is better to do in constructor or ngOnInit

Reply:
I didn't know about it, I'll change this in the next PR